cypress/e2e/helpers/surveyHelper.ts

```diff
@@ -46,14 +55,14 @@ export const SCHEDULE_TRIWEEKLY = (day: string, time: string) =>
 export const SCHEDULE_WEEKLY = (day: string, time: string) =>
   `Weekly on ${day} at ${time}`;
 
-export const verifySurveyCard = (
-  surveyName: string,
-  schedule: string,
-  status: string,
-) => {
+export const OPTION_RADIO_EXCELLENT = 'input[value="Excellent"]';
+export const OPTION_RADIO_GOOD = 'input[value="Good"]';
+export const OPTION_RADIO_CONFUSING = 'input[value="Confusing"]';
+
+export const currentDate = moment().format("MMM DD");
```

Comment:
Ok thanks, I removed it and put it in the test, it's because I thought that if I did it this way it would reduce the repetition of this variable in each test, what do you think?